build.sbt

```diff
@@ -19,6 +19,7 @@ libraryDependencies ++= {
   val scalaTestVersion = "3.0.0"
   val configVersion = "1.3.1"
   val json4sVersion = "3.5.0"
+  val iotHubServiceClientVersion = "1.0.11"
```

Comment:
the latest is 1.0.12, with a new namespace+name

"com.microsoft.azure.sdk.iot" % "iot-service-client" % "1.0.12"